app/lib/forms/teacher_reference_number.rb

```diff
@@ -12,17 +12,41 @@ def self.permitted_params
       ]
     end
 
+    # Required since this is the first question
+    # If it wasn't overridden it would check if any previous questions were answered
+    # and since there aren't any it would assume the user was trying to skip questions
+    # and redirect them to the start page
+    # Only overridden when this is the first question, which is now when the GAI pilot is on
+    # for the current user
+    def requirements_met?
+      return true if wizard.tra_get_an_identity_omniauth_integration_active?
+
+      super
+    end
+
     def next_step
       case trn_knowledge
       when "yes"
-        :contact_details
+        # This should never happen, as the button that would lead here should be replaced
+        # in this scenario with a link to GAI. However, just in case we have this fallback to take
+        # you to an interstitial page that can only go to GAI.
+        # This can happen if JS is disabled.
```

Comment:
Should this open with "This should only happen if JS is disabled, ..."?

Reply:
I’ll improve this comment. There are multiple ways to reach this state, they’re just obscure. I’ve catalogued them since writing this.